lib/common/errors.js

```diff
@@ -82,6 +82,12 @@ function ErrorFactory(util) {
     }
     util.inherits(TaskCancellationError, BaseError);
 
+    function TaskStopError(message) {
```

Comment:
@VulpesArtificem let's add a comment either here or in the Task.js code elaborating on this.